tests20/python_client/testcases/test_search.py

```diff
@@ -1,12 +1,666 @@
+import time
 import pytest
+import pandas as pd
+import random
+
 from base.client_request import ApiReq
 from utils.util_log import test_log as log
 from common.common_type import *
+from common.common_func import *
```

Comment:
do not import *, it would fail in code analysis check

Reply:
OK, modified.